client/stack-editor/lib/index.coffee

```diff
@@ -167,33 +176,33 @@ module.exports = class StackEditorAppController extends AppController
 
     confirmation = null
 
-    stackTemplate.on? 'update', =>
-
-      return  if confirmation
-      return  if view._stackSaveInAction
-
-      view.addSubView confirmation = new ContentModal
-        title: 'Stack Template is Updated'
-        cssClass: 'content-modal'
-        buttons:
-          cancel:
-            title: 'Cancel'
-            style: 'solid light-gray medium'
-            callback: ->
-              confirmation.destroy()
-              confirmation = null
-          ok:
-            title: 'OK'
-            style: 'solid green medium'
-            callback: =>
-              view.destroy()
-              delete @editors[stackTemplate._id]
-              @openEditor stackTemplate._id
-        content: '''
-          <div class='modalformline'
-            <p>This stack template is updated by another admin. Do you want to reload?</p>
-          </div>
-          '''
+    # stackTemplate.on? 'update', =>
```

Comment:
I commented out those lines because of #9527.
I think we need to get rid of those lines? @gokmen